### PR TITLE
Add sync pipeline for cost-management data

### PR DESCRIPTION
**Pull request by accorvin**
## Related Issues and Dependencies

None

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add required objects to sync the cost management data using Solgate

### CODE REVIEW

File: manifests/overlays/prod/cost-management/cost-management.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cost-management
```
Comment (tumido) [inline manifests/overlays/prod/cost-management/cost-management.enc.yaml:4]: This name needs to match `CronWorkflow`'s `config` parameter. In this case it needs to be `solgate-cost-management`.

**Review by tumido — CHANGES_REQUESTED** [review]
There's only one issue with this PR, otherwise it looks great! Thanks @accorvin  :slightly_smiling_face:

File: manifests/overlays/prod/cost-management/cost-management.enc.yaml (hunk 1 above)
Comment (accorvin) [reply manifests/overlays/prod/cost-management/cost-management.enc.yaml]: Fixed.

**Review by tumido — APPROVED** [review]
/lgmt
/approve

**Pull request merged**